src/components/UserInfo.js

```diff
@@ -1,14 +1,16 @@
 export default class UserInfo {
-    constructor(nameSelector, professionSelector) {
+    constructor(nameSelector, professionSelector, imageSelector) {
         this._name = document.querySelector(nameSelector);
         this._profession = document.querySelector(professionSelector);
+        this._image = document.querySelector(imageSelector);
     }
 
     // Retrieve profile info from the page
     getUserInfo() {
         return {
             name: this._name.textContent,
             profession: this._profession.textContent,
+            avatar: this._image.src
```

Comment:
The avatar is controlled by a different form. If you want to get it then you should create a separate method `getUserImage` (there is no need to get all the info with avatar when opening the profile form)

Reply:
That's been updated! Since I'm not getting the avatar when opening the profile form, I've removed it.